cmake/Config.cmake

```diff
@@ -19,8 +19,24 @@ if (WIN32)
             "${CMAKE_CURRENT_SOURCE_DIR}/cs/test/compat/bin/retail")
 endif()
 
-find_package (PythonLibs 2.7)
-find_package (PythonInterp 2.7)
+# find python interpreter and library
+# to specify a different version, invoke cmake with:
+# -DPython_ADDITIONAL_VERSIONS=Major.Minor
+find_package (PythonInterp ${Python_ADDITIONAL_VERSIONS})
```

Comment:
After fiddling with some of the options for cmake, I removed the tag altogether and this appears to be the proper way to configure python2 vs python3:
- Python2: `cmake`
- Python3: `cmake -DPYTHON_EXECUTABLE=/usr/bin/python3`

For any versions that cmake has no implicit built-in support, then the user can specify `Python_ADDITIONAL_VERSIONS`